### PR TITLE
bump ueberauth requirement

### PR DESCRIPTION
**Pull request by connorjacobsen**
- Bump Ueberauth requirement
- Update handling of state parameter for updated Ueberauth version

### CODE REVIEW

File: lib/ueberauth/strategy/linear.ex [hunk 1]
```diff
@@ -16,22 +17,19 @@ defmodule Ueberauth.Strategy.Linear do
...
     callback_url =
       if String.ends_with?(callback_url, "?"),
         do: String.slice(callback_url, 0..-2),
         else: callback_url
 
-    opts = Keyword.put(opts, :redirect_uri, callback_url)
+    opts =
+      []
+      |> with_scopes(conn)
+      |> with_state_param(conn)
```
Comment (OmegaNalphA) [inline lib/ueberauth/strategy/linear.ex:30]: hmm I dont see with_state_param declared, I mightve missed it is it in another file?

**Review by OmegaNalphA — COMMENTED** [review]
One q before approve since I think I missed something

File: lib/ueberauth/strategy/linear.ex (hunk 1 above)
Comment (connorjacobsen) [reply lib/ueberauth/strategy/linear.ex:30]: library function that is imported

Comment (OmegaNalphA) [reply lib/ueberauth/strategy/linear.ex:30]: ahhh

**Review by OmegaNalphA — APPROVED** [review]

File: lib/ueberauth/strategy/linear.ex (hunk 1 above)
Comment (connorjacobsen) [reply lib/ueberauth/strategy/linear.ex:30]: For posterity: https://github.com/ueberauth/ueberauth/blob/master/lib/ueberauth/strategies/helpers.ex#L193

File: lib/ueberauth/strategy/linear.ex [hunk 2]
```diff
@@ -41,6 +39,11 @@ defmodule Ueberauth.Strategy.Linear do
   Handles the callback from Linear.
   """
   def handle_callback!(%Plug.Conn{params: %{"code" => code}} = conn) do
+    req_headers = conn.req_headers
+    Logger.warn("req_headers: #{inspect(req_headers)}")
+    resp_headers = conn.resp_headers
+    Logger.warn("resp_headers: #{inspect(resp_headers)}")
```
Comment (connorjacobsen) [inline lib/ueberauth/strategy/linear.ex:45]: I need to delete this

**Pull request merged**